### DISCUSSION TITLE
Access to the name of the variant in the dashboard order sheet

### DISCUSSION THREAD

**Discussion by DidoMarchet** [saleor/saleor#11923] [General] [answered]
Hi guys,
our client ask us ho to say the attributes of the variants in the order sheet.
I try to fund them but actually there're no reference about the variant they sell.
There's the sku but it's a little bit annoying copy and paste it to find the variant into the catalogue.

The situation now is:
<img width="1076" alt="Screenshot 2023-02-06 at 14 31 10" src="https://user-images.githubusercontent.com/38940340/216984944-3db86e8d-b8ac-4b23-903a-9ec7c8274431.png">

So my question is if is possible to add more info about the variant (the name for example).

Thanks in advance and regards,

Davide

**Comment by timuric** [comment, marked as answer]
@DidoMarchet we are in the process of converting tables to datagrid which allows users to customize columns.

**Reply by DidoMarchet** [reply]
Great!
Thanks and kind regards,

Davide

**Comment by zanami** [comment]
Single order view also doesn't display variant names. If SKU isn't set there's not way to determine what variant it is. For example, Blue Long Sleeved Checked Shirt lacks the size information.
The variant name is present in a PDF invoice if you generate one, like this: Blue Long Sleeved Checked Shirt (M)
